### PR TITLE
add class container-fluid to fix bottom-scroll

### PR DESCRIPTION
**Pull request by dojutsu-user**
#126

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@dojutsu-user can you provide some live link from your fork and also resolve conflicts if you can.

**Comment by dojutsu-user** [comment]
@pradeepgangwar in the forked repo, i just tried to remove container-fluid class and now it works fine
https://dojutsu-user.github.io/OpenCode-Collaborative/

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```
Comment (pradeepgangwar) [inline _config.yml]: Remove this file. Not needed.

**Comment by dojutsu-user** [comment]
@pradeepgangwar  i am unable to resolve conflict.

**Comment by pradeepgangwar** [comment]
@dojutsu-user You index.html is outdated. Refer to current index.html and see. Pull latest changes from master and rebase your PR and make changes accordingly to new file.

**Comment by pradeepgangwar** [comment]
issue is closed. So closing this PR

**Pull request closed without merging**